app/src/main/java/org/mozilla/fenix/settings/account/SignOutFragment.kt

```diff
@@ -10,39 +10,36 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.FrameLayout
-import androidx.fragment.app.DialogFragment
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import org.mozilla.fenix.addons.runIfFragmentIsAttached
 import androidx.lifecycle.lifecycleScope
 import androidx.navigation.fragment.findNavController
 import com.google.android.material.bottomsheet.BottomSheetBehavior
-import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import androidx.appcompat.app.AppCompatDialogFragment
 import kotlinx.android.synthetic.main.fragment_sign_out.view.*
 import kotlinx.coroutines.launch
 import mozilla.components.service.fxa.manager.FxaAccountManager
 import org.mozilla.fenix.R
 import org.mozilla.fenix.ext.requireComponents
 
-class SignOutFragment : BottomSheetDialogFragment() {
+class SignOutFragment : AppCompatDialogFragment() {
     private lateinit var accountManager: FxaAccountManager
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setStyle(DialogFragment.STYLE_NO_TITLE, R.style.FirefoxAccountsDialogStyle)
+        setStyle(STYLE_NO_TITLE, R.style.BottomSheet)
```

Comment:
What does this style change do? Why are we using `BottomSheet` instead of the FxA styles?